recipes/openexr/2.4.0/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  "2.4.0":
+    url: "https://github.com/openexr/openexr/archive/v2.4.0.tar.gz"
+    sha256: "4904c5ea7914a58f60a5e2fbc397be67e7a25c380d7d07c1c31a3eefff1c92f1"
+patches:
```

Comment:
```suggestion
```

as this PR is to a very specific version, including the folder, and you don't need a patch, so you can remove it.